src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +11,32 @@
     private ReadWriteLock lock = new ReentrantReadWriteLock();
 
```

Comment:
you can add an initialization block for writeLock and readLock, but this is at your discretion)